influx-writer/influx-writer.py

```diff
@@ -15,29 +14,78 @@
 from time import time, sleep
 import config
 from listenstore import InfluxListenStore
+from requests.exceptions import ConnectionError
+from redis import Redis
+from redis_keys import INCOMING_QUEUE_SIZE_KEY, UNIQUE_QUEUE_SIZE_KEY
 
 REPORT_FREQUENCY = 5000
-SUBSCRIBER_NAME = "in"
-KEYSPACE_NAME_INCOMING = "ilisten"
-KEYSPACE_NAME_UNIQUE = "ulisten"
-
-
 DUMP_JSON_WITH_ERRORS = False
+ERROR_RETRY_DELAY = 3 # number of seconds to wait until retrying an operation
+
 
-class InfluxWriterSubscriber(RedisPubSubSubscriber):
-    def __init__(self, ls, influx, redis):
-        RedisPubSubSubscriber.__init__(self, redis, KEYSPACE_NAME_INCOMING, __name__)
+class InfluxWriterSubscriber(object):
```

Comment:
Fair enough. My point about an abstract writer class still stands. Other methods that can go in it are `callback`, `static_callback`, and `start`. Here's some more information about them and why they can be useful:

* https://www.python.org/dev/peps/pep-3119/
* http://stackoverflow.com/a/3571946/272770
* https://docs.python.org/2/library/abc.html